lib/after_layout.dart

```diff
@@ -3,11 +3,17 @@ library after_layout;
 import 'package:flutter/widgets.dart';
 
 mixin AfterLayoutMixin<T extends StatefulWidget> on State<T> {
+  bool _hasBeenCalled;
+
   @override
   void initState() {
     super.initState();
-    WidgetsBinding.instance
-        .addPostFrameCallback((_) => afterFirstLayout(context));
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      if(true != _hasBeenCalled) {
```

Comment:
this check looks so ugly

Reply:
Does the job. Feel free to fix the bug otherwise.

Reply:
Initstate() will only be called once

Reply:
@kalaganov what do you mean by `ugly`? You wanted to make it shorter like `if (!_hasBeenCalled) {}` or what?